IconToggle/index.d.ts

```diff
@@ -2,10 +2,15 @@ import MaterialComponent from '../MaterialComponent';
 import { VNode } from 'preact';
 import { MDCFoundation, MDCComponent, MDCRipple } from '../MaterialComponentsWeb';
 
+declare interface IconToggleState {
+  label?: string;
+  content?: string;
+  cssClass?: string;
+}
+
 declare interface IIconToggleProps extends JSX.HTMLAttributes {
```

Comment:
Missing the prop `onChange`.

Reply:
That is also inherited from `HTMLAttributes` already